src/main/java/uk/nhs/hee/tis/trainee/forms/dto/LtftSummaryDto.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2025 Crown Copyright (Health Education England)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.nhs.hee.tis.trainee.forms.dto;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.Builder;
+import org.bson.types.ObjectId;
+import uk.nhs.hee.tis.trainee.forms.dto.enumeration.LifecycleState;
+
+/**
+ * A summary of a LTFT application.
+ */
+@Builder
+public record LtftSummaryDto(
+    @JsonSerialize(using = ToStringSerializer.class)
+    ObjectId id,
+
+    String name,
+    UUID programmeMembershipId,
+    LifecycleState status,  //assumption: same as for FormR forms.
```

Comment:
Is it going to have the `APPROVED`, `WITHDRAWN` and `REJECTED` for LTFT status?

Reply:
Yeah, that would make sense. I am not sure if we want to set up a new enum for LTFT states, or extend the existing one we use for FormRs.

Reply:
We probably want to extend the existing enum, but may need to add some validation for valid statuses per form type

Reply:
Happy to build that in now, unless we'd prefer to keep this PR from getting any bigger?

Reply:
I don't think we need to do that just yet, we can add those statuses as part of the appropriate tickets that will enable that functionality.